app.rb

```diff
@@ -18,194 +18,288 @@
   Dotenv.load
 end
 
-class Tint < Sinatra::Base
-  helpers Sinatra::Streaming
+module Tint
+  PROJECT_PATH = ENV["PROJECT_PATH"]
 
-  configure :development do
-    register Sinatra::Reloader
-  end
+  class App < Sinatra::Base
```

Comment:
Oh, I see.  We just moved it down so that more things can live in our namespace.  Ok.